app/main/views/g_cloud.py

```diff
@@ -556,6 +558,7 @@ def view_project(framework_family, project_id):
         following_framework=following_framework,
         project=project,
         custom_dimensions=custom_dimensions,
+        count=search_summary_count,
```

Comment:
It might be nicer to pass the entire `search_meta` into the template rather than the two pieces? I know that's a bit more refactoring but feels a bit more future-proof?

Reply:
I agree with what you're saying in principle, but in the interests of keeping this a quick fix I'm going to leave it as is.

I'm also not sure that search_meta is the best place to be getting the count from; the code around it and `SearchSummary` is quite convoluted and I think refactoring them should definitely be done, but not in this PR.